client/reader/start/card.jsx

```diff
@@ -0,0 +1,31 @@
+// External dependencies
+import React from 'react';
+
+// Internal dependencies
+import Card from 'components/card';
+import SiteIcon from 'components/site-icon';
+import FollowButton from 'reader/follow-button';
+import StartPostPreview from './post-preview';
+
+const StartCard = React.createClass( {
```

Comment:
Maybe just call these SiteCard's? I could totally see using them in the Site recs section.

Maybe they should go in components too?